opentracing/standardtracer/standardimpl.go

```diff
@@ -41,6 +41,16 @@ func (s *standardSpan) SetTag(key string, value interface{}) opentracing.Span {
 	return s
 }
 
+func (s *standardSpan) SetTags(tags opentracing.Tags) opentracing.Span {
+	s.lock.Lock()
+	defer s.lock.Unlock()
+
+	for k, v := range tags {
+		s.raw.Tags[k] = fmt.Sprint(v)
```

Comment:
at one point, though I had some cold feet when I started to codify it... felt like it was making the simple case un-simple (and it'll be easier to add repeated tags later than to remove them).